prow/pj-on-kind.sh

```diff
@@ -79,7 +79,7 @@ function ensureInstall() {
   # Install kind and set up cluster if not already done.
   if ! command -v kind >/dev/null 2>&1; then
     echo "Installing kind..."
-    GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0
+    GO111MODULE="on" go get sigs.k8s.io/kind@v0.16.0
```

Comment:
what's this used for?

Reply:
it should be `go install` , no?

Reply:
possibly, it depends on the go version, for a long time go supported both.
this is used when the user doesn't have kind installed.

the script is used to run prowjobs locally on kind for job author devel